src/series/bar.js

```diff
@@ -26,12 +33,12 @@
                     .width(width)
                     .height(0);
 
+                pathGenerator.height(0);
```

Comment:
No worries, I've got some other things I want to be deplore in parallel.

And good luck for tomorrow!